source/clear-linux/tutorials/dlaas-benchmarks.rst

```diff
@@ -0,0 +1,186 @@
+
+Download and run either the `TensorFlow`_ or the `TensorFlow with MKL-DNN`_  docker image from hub.docker.com. The next commands will take place in the running container. Replace <docker_name> with the name of the image.
+
+First, clone the benchmark repository
+
+.. code-block:: bash
+
+  docker exec -t <docker_name> bash -c ‘git clone http://github.com/tensorflow/benchmarks -b cnn_tf_v1.11_compatible’
+
+Next, execute the benchmark script to run the benchmark
+
+.. code-block:: bash
+
+  docker exec -i <docker_name> bash -c ‘python benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py --device=cpu --model=resnet50 --data_format=NWHC ’.
+
+You can replace the model with one of your choice supported by the TensorFlow benchmarks.
+
+
+
+Kubeflow Benchmarks
+===================
+
+The benchmark workload will run in a Kubernetes (k8s) container. We will use Kubeflow and deploy three nodes for this tutorial, to get a decent return.
+
+k8s setup
++++++++++
+
+Please follow the instructions in the :ref:`kubernetes` tutorial to get set up on |CL|.  The k8s community also has `instructions`_ if you are using Ubuntu instead of |CL|.
+
+
+k8s networking
+++++++++++++++
+
+We have used `flannel`_ as the network provider for these tests. If you are comfortable with another network layer, there is nothing that will prevent you from using that.
+Refer to the Kubernetes `networking documentation`_ for setup.
+
+
+Images
+++++++
+
+We will need to create a docker image that will include launcher.py to include the |CL| Deep Learning Stack, and put the benchmarks repo in the right location. From the docker image, run the following
+
+.. code-block:: bash
+
+  mkdir -p /opt
+  git clone https://github.com/tensorflow/benchmarks.git /opt/tf-benchmarks
+  cp launcher.py /opt
+  chmod u+x /opt/*
+
+Your entry point then becomes "/opt/launcher.py".
+
+
+This will build an image which can be consumed directly by TFJob from kubeflow.  We are working to create these images as part of our release cycle.
+
+
+Ksonnet
++++++++
+
+Kubeflow uses Ksonnet to manage deployments, so we need to install that before setting up Kubeflow. On |CL|, follow these steps:
+
+.. code-block:: bash
+
+  swupd bundle-add go-basic-dev
+  export GOPATH=$HOME/go
+  export PATH=$PATH:$GOPATH/bin
+  go get github.com/ksonnet/ksonnet
+  cd $GOPATH/src/github.com/ksonnet/ksonnet
+  make install
+
+
+.. For other OS, follow steps in: https://www.kubeflow.org/docs/guides/components/ksonnet/ .
+
+After the ksonnet installation is complete, ensure that binary `ks` is accessible across the environment.
+
+
+Kubeflow
+========
+Once you have k8s running on your nodes, you can setup Kubeflow by following these instructions from their `quick start guide`_.
+
+.. code-block:: bash
+
+  export KUBEFLOW_SRC=$HOME/kflow
```

Comment:
```suggestion
export KUBEFLOW_SRC=$HOME/kflow
export KUBEFLOW_TAG=”v0.3.2”
export KFAPP=”kflow_app”
export K8S_NAMESPACE=”kubeflow”

mkdir ${KUBEFLOW_SRC}
cd ${KUBEFLOW_SRC}
ks init ${KFAPP}
cd ${KFAPP}
ks registry add kubeflow github.com/kubeflow/kubeflow/tree/${KUBEFLOW_TAG}/kubeflow
ks pkg install kubeflow/core
```

Reply:
@mvincerx could you please tell me where this change was applied? I see no change on the last commit.

Reply:
@DnPlas. Se ve todos los cambiamentos ahora. Todas las lineas han cambiado como resultado del mi commit antes.

Reply:
@mvincerx Gracias! Ahora veo los cambios.